build/minikube.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Dashboard Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Base directory, that is cached.
+BASE_DIR=.k8s
+sudo mkdir -p ${BASE_DIR}
+
+# Version heapster to use.
+HEAPSTER_VERSION="v1.4.0"
+# Port of the heapster to serve on.
+HEAPSTER_PORT=8082
+
+# Latest stable minikube version.
+MINIKUBE_VERSION=v0.22.3
+
+# Latest stable Kubernetes version.
+K8S_VERSION=$(curl -sSL https://dl.k8s.io/release/stable.txt)
+
+# Binaries location.
+KUBECTL_BIN=${BASE_DIR}/kubectl-${K8S_VERSION}
+MINIKUBE_BIN=${BASE_DIR}/minikube-${MINIKUBE_VERSION}
+
+# TODO Check if it doesn't exist in cache.
+echo "Downloading minikube ${MINIKUBE_VERSION}"
+curl -L https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64 -o ${MINIKUBE_BIN}
+chmod +x ${MINIKUBE_BIN}
+${MINIKUBE_BIN} version
+
+# TODO Check if it doesn't exist in cache.
+echo "Downloading kubectl ${K8S_VERSION}"
+curl -L https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl -o ${KUBECTL_BIN}
+chmod +x ${KUBECTL_BIN}
+${KUBECTL_BIN} version --client
+
+echo "Setting up minikube"
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=${BASE_DIR}
+export CHANGE_MINIKUBE_NONE_USER=true
+
+echo "Setting up environment"
+mkdir -p $HOME/.kube
+touch $HOME/.kube/config
+
+echo "Starting Kubernetes ${K8S_VERSION}"
+sudo -E ${MINIKUBE_BIN} start --vm-driver=none
+
+${KUBECTL_BIN} proxy -p 8080 &
```

Comment:
hmm..why is this needed?

Reply:
It's not. Leftover from deploying heapster with kubectl.